gms/gossiper.cc

```diff
@@ -1550,7 +1550,7 @@ void gossiper::mark_alive(inet_address addr, endpoint_state& local_state) {
     // Enter the _background_msg gate so stop() would wait on it
     auto gh = _background_msg.hold();
     logger.debug("Sending a EchoMessage to {}, with generation_number={}", id, generation);
-    (void)_messaging.send_gossip_echo(id, generation.value(), std::chrono::milliseconds(15000)).then([this, addr] {
+    (void)_messaging.send_gossip_echo(id, generation.value(), std::chrono::milliseconds(15000)).then([this, addr] () -> future<> {
```

Comment:
Some of the changes in this patch, like this one and the other one below, aren't really relevant and I'm not sure if they are justified.
Perhaps if you want them, you can move them to a separate patch with some excuse why you want them?

Reply:
sorry, debugging leftovers, will remove hunk.